### DISCUSSION TITLE
FreeRTOS?

### DISCUSSION THREAD

**Discussion by paul-ratazzi** [eugene-tarassov/vivado-risc-v#215] [General]
Hello and thank you for this excellent work.  I am interested in running FreeRTOS (or similar) instead of Debian and noticed you mention this possibility in the readme. Has there been any effort in this direction? Is it a matter of building a different SD image?

**Comment by eugene-tarassov** [comment]
I only provide ready-to-use script to build Debian SD card image. To use any other OS, you need similar script for that OS. Also, you will need to port SD, Ethernet and UART device drivers. I don't know if anybody has already done that.
